Documentation/op-guide/security.md

```diff
@@ -426,6 +426,9 @@ Make sure to sign the certificates with a Subject Name the member's public IP ad
 
 The certificate needs to be signed for the member's FQDN in its Subject Name, use Subject Alternative Names (short IP SANs) to add the IP address. The `etcd-ca` tool provides `--domain=` option for its `new-cert` command, and openssl can make [it][alt-name] too.
 
+### etcd is insecure by default
+Note that etcd doesn't enforce users to use the authentication feature in the transport layer or [RBAC based authentication][auth]. It means an etcd cluster which doesn't enable the security features can expose its data to any clients. Users need to enable the features for avoiding such a situation.
```

Comment:
@mitake thanks, this looks great but just a thought, if you and other reviewers like, -- how about putting this at the beginning of the document? Something like in bold in here "etcd supports automatic TLS as well as authentication through client certificates for both clients to server as well as peer (server to server / cluster) communication. **Note that these security features are not enabled by the default installation of etcd. To secure an etcd cluster and not expose its data to any clients, users nust enable one or more security features as described in this document.** "

Reply:
I think context on WHY etcd doesn’t have authentication, authorization, or TLS on by default is important as well: “etcd doesn’t enable auth or transport security by default to reduce friction for users getting started with the database. Further, changing this default would be a breaking change for the project which was established 7 years ago.”

Reply:
@philips thanks for the insight, that's great point. Earlier "by design" was the only thing that we discussed for this doc update.

Reply:
Thanks for your comments @spzala and @philips ! I updated this PR based on the comment. Could you take a look at the latest version?